### PR TITLE
Update brandings.css

### PR DESCRIPTION
**Pull request by jgjimenez**
Login page's labels get cut off in certain languages

### CODE REVIEW

File: root/usr/share/cockpit/branding/nethserver/branding.css [hunk 1]
```diff
@@ -220,3 +220,23 @@ input[type="checkbox"].form-control {
...
+}
+
+input[type="text"].form-control { 
+width: 90%;
+margin-left: auto;
+}
+input[type="password"].form-control {
+width: 90%;
+margin-left: auto;
+}
```
Comment (gsanchietti) [inline root/usr/share/cockpit/branding/nethserver/branding.css:241]: For sure these selector are way too wide: you're going to change almost every form in the UI.
You should narrow the scope of the selectors.

**Review by gsanchietti — CHANGES_REQUESTED** [review]

File: root/usr/share/cockpit/branding/nethserver/branding.css (hunk 1 above)
Comment (jgjimenez) [reply root/usr/share/cockpit/branding/nethserver/branding.css:241]: I am going to leave it without effect, what is happening is not css problems, by increasing the resolution of my screen it is possible to see well.

Comment (jgjimenez) [reply root/usr/share/cockpit/branding/nethserver/branding.css:241]: it is not necessary to apply it for production

**Comment by gsanchietti** [comment]
Closed as per request of @jgjimenez

**Pull request closed without merging**